p2p/node/pubsubManager/gossipsub.go

```diff
@@ -395,6 +397,8 @@ func (g *PubsubManager) Broadcast(location common.Location, data interface{}) er
 		return err
 	}
 	if value, ok := g.topics.Load(topicName.String()); ok {
+		mu.Lock()
```

Comment:
Why is there a lock here? Did it previously cause a race?

Reply:
Good catch, shouldn't be here nor a global. Moved it to the test